### PR TITLE
Add Signal Task Templates

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**

### CODE REVIEW

File: Signals/resources/catalog/Send_Signal.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <variable name="SIGNAL" value="stop" inherited="false" />
+      </variables>
+      <genericInformation>
+        <info name="task.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/signal-send.png"/>
+      </genericInformation>
+      <scriptExecutable>
+        <script>
+          <code language="groovy">
+            <![CDATA[
+signalapi.addSignal(variables.get("SIGNAL"))
```
Comment (fviale) [inline Signals/resources/catalog/Send_Signal.xml:33]: just saw this, sorry I didn't mention it earlier. I would have prefered sendSignal rather then addSignal

**Comment by fviale** [comment]
@nebilbenmabrouk why only workflows ? We could also have some pre and post scripts (referenced enventually by these workflows). 
Something like 

wait_for_signal.groovy (1 argument):
signal = args[0]
println("Waiting for signal "+ signal)
signalapi.waitFor(signal)

ready_for_signal.groovy (1 argument):
signal = args[0]
println("Ready for signal "+ signal)
signalapi.readyForSignal(signal)

File: Signals/resources/catalog/Send_Signal.xml (hunk 1 above)
Comment (nebilbenmabrouk) [reply Signals/resources/catalog/Send_Signal.xml:33]: OK, I'll do 2 new PRs one for scheduling and one for proactive-examples. Thanks

**Comment by amouhoub** [comment]
@nebilbenmabrouk You must add the following line to `build.gradle` in order for your bucket to be included
```include 'Signals/**'```
You must also add the bucket name to the `ordered_bucket_list` file as well

File: Signals/METADATA.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	"metadata": {
+		"slug": "signals",
+		"name": "signals",
+		"short_description": "ProActive task templates for signal management",
+		"author": "ActiveEon's Team",
+		"tags": ["signal", "template", "action", "communication"],
+		"version": "1.0"
+	},
+	"catalog" : {
+		"bucket" : "Signal",
```
Comment (amouhoub) [inline Signals/METADATA.json:11]: @nebilbenmabrouk bucket name should be all lowercase, even here

**Review by amouhoub — CHANGES_REQUESTED** [review]

File: Signals/METADATA.json (hunk 2 above)
Comment (nebilbenmabrouk) [reply Signals/METADATA.json]: Done :)

**Pull request merged**